skops/io/tests/test_persist.py

```diff
@@ -221,6 +221,15 @@ def _tested_estimators(type_filter=None):
     )
 
 
+def _tested_ufuncs():
+    for ufunc_name in SCIPY_UFUNC_TYPE_NAMES:
+        parts = ufunc_name.split(".")
+        module_name = ".".join(parts[:-1])
+        ufunc_name = parts[-1]
+
+        yield gettype(module_name=module_name, cls_or_func=ufunc_name)
```

Comment:
```suggestion
    for full_name in SCIPY_UFUNC_TYPE_NAMES:
        module_name, _, ufunc_name = full_name.rpartition(".")
        yield gettype(module_name=module_name, cls_or_func=ufunc_name)
```

Just a suggestion to make the code more concise ;)

Reply:
Adapted!